src/praisonai/tests/unit/test_autogen_v4_integration.py

```
@@ -0,0 +1,436 @@
+        # Mock the v0.4 components
+        mock_model_client = AsyncMock()
+        mock_assistant = Mock()
+        mock_group_chat = AsyncMock()
+        mock_result = Mock()
+        mock_result.messages = [Mock(content="Test completion result")]
+        mock_group_chat.run.return_value = mock_result
+        
+        with patch('praisonai.praisonai.agents_generator.OpenAIChatCompletionClient', return_value=mock_model_client), \
+             patch('praisonai.praisonai.agents_generator.AutoGenV4AssistantAgent', return_value=mock_assistant), \
+             patch('praisonai.praisonai.agents_generator.RoundRobinGroupChat', return_value=mock_group_chat), \
+             patch('praisonai.praisonai.agents_generator.TextMentionTermination') as mock_text_term, \
```

Ruff (0.12.2)

Local variable `mock_text_term` is assigned to but never used

Remove assignment to unused variable `mock_text_term`

(F841)